site2/website/release-notes.md

```diff
@@ -1,6 +1,419 @@
+- Fix transaction timeout issue at the client side [#10323](https://github.com/apache/pulsar/pull/10323)
+- Fix transaction client reconnect issue after transaction coordinator unloaded [#10327](https://github.com/apache/pulsar/pull/10327)
+- Fix transaction timeout not canceled after expired [#10366](https://github.com/apache/pulsar/pull/10366)
+- Transaction log low watermark optimization [#10422](https://github.com/apache/pulsar/pull/10422)
+- Fix transaction buffer client channel not active problem [#10407](https://github.com/apache/pulsar/pull/10407)
+- Admin API for getting transaction coordinator stats [#10639](https://github.com/apache/pulsar/pull/10639)
+- Admin API for getting transaction in transaction buffer stats [#10642](https://github.com/apache/pulsar/pull/10642)
+- Admin API for getting transaction in pending ack stats [#10648](https://github.com/apache/pulsar/pull/10648)
+- Admin API for getting transaction buffer stats and pending ack stats [#10650](https://github.com/apache/pulsar/pull/10650)
+- Admin API for getting transaction coordinator internal stats [#10653](https://github.com/apache/pulsar/pull/10653)
+- Setup transaction metadata with metadata store [#10677](https://github.com/apache/pulsar/pull/10677)
+- Fix issue with acknowledge messages from multiple subscriptions of a topic [#10689](https://github.com/apache/pulsar/pull/10689)
+- Admin API for getting transaction metadata [#10690](https://github.com/apache/pulsar/pull/10690)
+- Admin API for getting slow transactions [#10701](https://github.com/apache/pulsar/pull/10701)
+- Fix transaction log handle managed ledger WriteFail state [#10711](https://github.com/apache/pulsar/pull/10711)
+- Admin API for getting pending ack internal stats [#10725](https://github.com/apache/pulsar/pull/10725)
+- Fix transaction ack delete marker position when don't have transaction ack [#10741](https://github.com/apache/pulsar/pull/10741)
+- Fix transaction pending ack generate managedLedgerStats fail exception [#10797](https://github.com/apache/pulsar/pull/10797)
+- Use zookeeper prometheus metric provider to export zookeeper metrics [#10803](https://github.com/apache/pulsar/pull/10803)
+
+#### Security
+- Optional auth method name header in http authentication [#6799](https://github.com/apache/pulsar/pull/6799)
+- Topics level authorization support [#7523](https://github.com/apache/pulsar/pull/7523)
+- Authorization for setting topic/subscription auto-creation and subscription expire time on namespace [#7900](https://github.com/apache/pulsar/pull/7900)
+- Allow serializable stream-provider field into AuthenticationTls [#10020](https://github.com/apache/pulsar/pull/10020)
+- Add configuration for running OWASP Dependency Check for all modules [#10288](https://github.com/apache/pulsar/pull/10288)
+- Update default TLS protocols to TLSv1.3,TLSv1.2 for the broker and proxy [#10598](https://github.com/apache/pulsar/pull/10598)
+
+#### Broker
```

Comment:
I think it is worth adding this PR to the broker's list of changes: https://github.com/apache/pulsar/pull/10223. It introduces a deprecated field as well as the preferred field. No fields will be removed until 3.0.0 (presumably), but I think we should advertise the change/deprecation here.

Reply:
@michaeljmarshall Added in both broker and client.